### PR TITLE
[IT] Add `How you can help!`

### PR DESCRIPTION
**Pull request by marcostudios**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Community/How_you_can_help!/it.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+Di seguito sono riportati alcuni dei modi con cui puoi aiutare la comunità:
+
+## Aiutaci con la wiki
+
+[La wiki](https://github.com/ppy/osu-wiki) è la nostra documentazione ufficiale che contiene tutto quello che riguarda osu!. E in quanto tale, la wiki è sempre alla ricerca di nuovi collaboratori per aiutare a mantenere il sito aggiornato con nuove, fresche conoscenze o traduzioni.
+
+Se sei nuovo nella osu! wiki, puoi leggere di più a riguardo nella [Guida per contibuire alla osu! wiki](/wiki/osu!_wiki/Contribution_guide).
+
+## Aiutaci a mantenere osu! amichevole e sicura
```
Comment (Sclafus) [inline wiki/Community/How_you_can_help!/it.md:15]: why is it feminine?

File: wiki/Community/How_you_can_help!/it.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+Tutti amano migliorarsi e necessitano di feedback. Spesso gli utenti chiedono feedback sui loro progressi nel gameplay nel [forum Gameplay & Rankings](https://osu.ppy.sh/community/forums/13) e altri attendono un feedback sulle loro ultime skin create nel [forum Work In Progress Skins](https://osu.ppy.sh/community/forums/119).
+
+## Aiuta o unisciti a progetti gestiti dagli utenti
+
+osu! ha una comunità fiorente di persone tecniche e competitive. Variano da hobbisti a veterani dei rhythm games. Ci sono molte proposte e progetti sviluppati esclusivamente per osu! e tu puoi aiutarli, o guidare il futuro di osu! con le tue proposte/progetti!
+
+Ecco alcuni dei modi con cui puoi contribuire:
+
+- Sfoglia il forum [Tornei](https://osu.ppy.sh/community/forums/55). I tornei richiedono molto tempo e lavoro per organizzarli senza alcun intoppo, tu puoi aiutarli facendo da giudici spettatori in certi casi.
+- Guada cosa succede nel forum [Development](https://osu.ppy.sh/community/forums/2); quì è dove si svolgono la maggiorparte dei progetti seri.
```
Comment (Sclafus) [inline wiki/Community/How_you_can_help!/it.md:64]: typo "Guada" -> "Guarda"

File: wiki/Community/How_you_can_help!/it.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+Tutti amano migliorarsi e necessitano di feedback. Spesso gli utenti chiedono feedback sui loro progressi nel gameplay nel [forum Gameplay & Rankings](https://osu.ppy.sh/community/forums/13) e altri attendono un feedback sulle loro ultime skin create nel [forum Work In Progress Skins](https://osu.ppy.sh/community/forums/119).
+
+## Aiuta o unisciti a progetti gestiti dagli utenti
+
+osu! ha una comunità fiorente di persone tecniche e competitive. Variano da hobbisti a veterani dei rhythm games. Ci sono molte proposte e progetti sviluppati esclusivamente per osu! e tu puoi aiutarli, o guidare il futuro di osu! con le tue proposte/progetti!
+
+Ecco alcuni dei modi con cui puoi contribuire:
+
+- Sfoglia il forum [Tornei](https://osu.ppy.sh/community/forums/55). I tornei richiedono molto tempo e lavoro per organizzarli senza alcun intoppo, tu puoi aiutarli facendo da giudici spettatori in certi casi.
```
Comment (Sclafus) [inline wiki/Community/How_you_can_help!/it.md:63]: The second part of the phrase looks odd to me. I would do something like:
"I tornei richiedono molto tempo e lavoro per essere organizzati senza alcun intoppo. Puoi aiutare facendo da giudice spettatore in determinati casi."

File: wiki/Community/How_you_can_help!/it.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+- Unisciti ai canali  `#mapping` e `#modreqs` nella chat in gioco e unisciti al discorso o semplicamente leggi cosa viene scritto.
+- Se hai tempo e dedizione, apri una [Modding Queue](https://osu.ppy.sh/community/forums/60) per dare uno spazio ai creatori di beatmap un posto per richiedere delle mods.
+
+## Dai feedback sui progressi dei giocatori o sulle skins!
+
+Tutti amano migliorarsi e necessitano di feedback. Spesso gli utenti chiedono feedback sui loro progressi nel gameplay nel [forum Gameplay & Rankings](https://osu.ppy.sh/community/forums/13) e altri attendono un feedback sulle loro ultime skin create nel [forum Work In Progress Skins](https://osu.ppy.sh/community/forums/119).
+
+## Aiuta o unisciti a progetti gestiti dagli utenti
+
+osu! ha una comunità fiorente di persone tecniche e competitive. Variano da hobbisti a veterani dei rhythm games. Ci sono molte proposte e progetti sviluppati esclusivamente per osu! e tu puoi aiutarli, o guidare il futuro di osu! con le tue proposte/progetti!
```
Comment (Sclafus) [inline wiki/Community/How_you_can_help!/it.md:59]: rhythm games -> "giochi ritmici"

**Review by Sclafus — APPROVED** [review]
LGTM

**Review by Sclafus — DISMISSED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**